enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/PropertyExistenceEnforcer.java

```diff
@@ -71,17 +71,17 @@ private PropertyExistenceEnforcer( List<LabelSchemaDescriptor> nodes, List<Relat
         this.relationshipConstraints = rels;
         for ( LabelSchemaDescriptor constraint : nodes )
         {
-            update( mandatoryNodePropertiesByLabel, constraint.getLabelId(), constraint.getPropertyIds() );
+            update( mandatoryNodePropertiesByLabel, constraint.getLabelId(), copyIds( constraint.getPropertyIds() ) );
         }
         for ( RelationTypeSchemaDescriptor constraint : rels )
         {
-            update( mandatoryRelationshipPropertiesByType, constraint.getRelTypeId(), constraint.getPropertyIds() );
+            update( mandatoryRelationshipPropertiesByType, constraint.getRelTypeId(),
+                    copyIds( constraint.getPropertyIds() ) );
         }
     }
 
     private static void update( PrimitiveIntObjectMap<int[]> map, int key, int[] values )
     {
-        Arrays.sort( values );
```

Comment:
As `values` now is assumed to be sorted, it would be nice to either
a) add an assert to check that it's sorted
or at least
b) rename it into `sortedValues` or similar